### DISCUSSION TITLE
如何在安卓系统部署歇斯底里啊？

### DISCUSSION THREAD

**Discussion by Mszcs-yy** [apernet/hysteria#814] [Q&A]
PC上面配置好了，手机上如何配置，singbox一启动就闪退

**Comment by haruue** [comment]
这里有一个客户端列表， 你可以先试试 singbox 以外的客户端能不能正常使用。
https://hysteria.network/zh/docs/getting-started/3rd-party-apps/

此外， 第三方应用的闪退问题， 你要反馈给第三方应用的开发者， 而不是反馈到这里。

**Reply by Mszcs-yy** [reply]
感谢你的回复，singbox出错就会闪退，nekobox给出了错误，我修正了错误之后在singbox和nekobox都成功搭建了
